consultation_analyser/support_console/jinja2/support_console/consultations/show.html

```diff
@@ -126,16 +132,20 @@ <h2 class="govuk-heading-m">Generate themes</h2>
         }) }}
 
         <h2 class="govuk-heading-m">Download JSON</h2>
-        {% if total_themes %}
-          <p class="govuk-body">This consultation has themes, so the JSON will be in the <a class="govuk-link" href="/schema/consultation_with_responses_and_themes_schema.json">ConsultationWithResponsesAndThemes</a> format. It will contain the themes generated most recently.</p>
-        {% else %}
-          <p class="govuk-body">This consultations has no themes, this JSON will be in the <a class="govuk-link" href="/schema/consultation_with_responses_schema.json">ConsultationWithResponses</a> format.</p>
-        {% endif %}
+        <p class="govuk-body">This consultation can be downloaded without themes as a JSON in the <a class="govuk-link" href="/schema/consultation_with_responses_schema.json">ConsultationWithResponses</a> format.</p>
         {{ govukButton({
-          'text': "Download JSON",
+          'text': "Download JSON (without themes)",
           'name': "download_json"
         }) }}
 
+        {% if total_themes %}
```

Comment:
Do we need this one if we have the download buttons below?

Reply:
Thanks, yes we can get rid of the extra button.